core/src/test/scala/org/apache/spark/serializer/KryoSerializerSuite.scala

```diff
@@ -242,6 +244,15 @@ class KryoSerializerSuite extends FunSuite with SharedSparkContext {
       ser.newInstance().deserialize[ClassLoaderTestingObject](bytes)
     }
   }
+
+  test("registration of HighlyCompressedMapStatus") {
+    val conf = new SparkConf(false)
+    conf.set("spark.kryo.registrationRequired", "true")
+    val hcmo = HighlyCompressedMapStatus(BlockManagerId("exec-1", "host", 1234), Array(0l,2l,5l))
```

Comment:
actually here. again style nit, add a space after comma for the arrays. and use capital L rather than lowercase l because it is hard to tell the difference between lowercase l and capital L